### DISCUSSION TITLE
Unique email per customer group.

### DISCUSSION THREAD

**Discussion by jramoyo-serenade** [swellstores/community#382] [Help] [answered]
We've created customer groups following the approach documented here:
https://developers.swell.is/guides/build-a-marketplace/create-account-hierarchies

If we ever sign-up a "buyer" using an email address, then later on, the same email is used to sign-up as a "vendor", it won't work because the email address should be unique.

Is there a way for us to relax this validation, so that emails are only unique per customer group?

**Comment by ericingram** [comment, marked as answer]
@jramoyo-serenade Customer groups are exclusive, which means an email can only belong to one group. We've planned to introduce a new concept called Segments which are non-exclusive, but it hasn't been prioritized yet.

I would suggest using a different mechanism if you need accounts to belong to multiple groups, perhaps a custom field
